### ISSUE TITLE
Clarifying about atom.io hosted packages during/after the sunset 

### ISSUE THREAD

**Issue by DeeDeeG** [atom/atom#25504]
Hi official Atom folks.

Sad to see [the editor is being sunset](https://github.blog/2022-06-08-sunsetting-atom/). **_But that's not what this post is about_.**

**The point of this issue is to ask**: Will package downloads (from https://atom.io/packages) continue to work after the sunset? (Effectively: Will a user with an Atom app on their system still be able to download/install addons packages?) Or will package downloads stop working?

Just wondering, as packages weren't mentioned in the announcement. I might have some follow-up questions, but that is the main question.

Thank you for openly communicating about this.

Best,
\- DeeDeeG

**Comment by vprigent** [comment]
Might be easier to sunset but would have been so much nicer to just help the few keen developers take over and keep this running for all users

**Comment by aalbacetef** [comment]
> but would have been so much nicer to just help the few keen developers take over and keep this running for all users

Yep, I would've been happy to jump on this if there was at least some form of onboarding.

**Comment by stech11845** [comment]
I think no because Atom is sunsetting...

**Comment by ItsAleph** [comment]
At least we have 6 months to download everything we need...
